x-pack/test/tsconfig.json

```diff
@@ -106,7 +106,6 @@
     "@kbn/journeys",
     "@kbn/stdio-dev-helpers",
     "@kbn/alerting-api-integration-helpers",
-    "@kbn/securitysolution-ecs",
```

Comment:
Clarification: is this being removed since only the child `security_solution_api_integration` now references it?